web/client/plugins/SaveAs.jsx

```diff
@@ -157,24 +161,17 @@ module.exports = {
         onMapSave: createMap,
         loadMapInfo,
         editMap,
+        resetCurrentMap,
         onDisplayMetadataEdit,
         onCreateThumbnail: createThumbnail
-
     })(assign(SaveAs, {
         BurgerMenu: {
             name: 'saveAs',
             position: 900,
             text: <Message msgId="saveAs"/>,
             icon: <Glyphicon glyph="floppy-open"/>,
-            action: editMap.bind(null, {}),
-            // display the BurgerMenu button only if the map can be edited
-            selector: (state) => {
```

Comment:
Because Lorenzo Natali wanted to display the SaveAsPlugin in every map not only in the empty project